docs/pet-kata/slides.md

```diff
@@ -1014,7 +1019,9 @@ Partition pet and non pet people
 @Test
 public void partitionPetAndNonPetPeople()
 {
-  PartitionMutableList<Person> partitionMutableList = this.people.partition(eachPerson -> eachPerson.isPetPerson());
+  PartitionMutableList<Person> partitionMutableList = this.people
+    .partition(eachPerson -> eachPerson.isPetPerson());
```

Comment:
This could be a method reference.  `Person::isPetPerson`

Reply:
done